examples/ui5-ts-app/test/e2e/invalidate-cache.test.ts

```diff
@@ -0,0 +1,44 @@
+// only requiring the service for late inject/init
+import Input from "sap/ui/webc/main/Input"
+import { default as _wdi5 } from "wdio-ui5-service"
+const wdi5 = new _wdi5()
+
+describe("invalidate cache", () => {
+    it("receives control and invalidates the cache after reloading the application", async () => {
+        // stores the control in the cache
+        const webcomponentValue = await (
+            browser.asControl({
+                selector: {
+                    bindingPath: {
+                        modelName: "",
+                        propertyPath: "/Customers('TRAIH')/ContactName"
+                    },
+                    viewName: "test.Sample.tsapp.view.Main",
+                    controlType: "sap.ui.webc.main.Input"
+                }
+            }) as unknown as Input
+        ).getValue()
+
+        expect(webcomponentValue).toEqual("Helvetius Nagy")
+
+        // reload the app
+        await browser.url("http://localhost:8080/index.html")
+        // inject ui5 again which should invalidate the cache
+        await wdi5.injectUI5()
```

Comment:
actually the test would fail if the invalidation would not work. Because we reload the application the DOM element which is stored in the cache is not valid anymore and following error would be thrown:

```bash
The element with reference ELEMENT-543 is stale; either the element is no longer attached to the DOM, it is not in the current frame context, or the document has been refreshed
stale element reference: stale element reference: The element with reference ELEMENT-543 is stale; either the element is no longer attached to the DOM, it is not in the current frame context, or the document has been refreshed
```